larq/callbacks.py

```diff
@@ -52,20 +64,32 @@ def set_model(self, model: keras.models.Model):
                 f'Optimizer must have a "{self.hyperparameter}" attribute.'
             )
 
-    def on_epoch_begin(self, epoch: int, logs: Optional[Dict] = None):
+    def set_hyperparameter(self, t: int):
```

Comment:
What about renaming `t` to `epoch_or_step`?

Reply:
I don't like that solution, seems a hideous variable name. I get your point, though I don't see any better alternatives either.

Reply:
Awesome. I just made some final changes, should be good to merge now.